src/views/InternalApp/ThePlans/components/PlanForm.vue

```diff
@@ -141,12 +136,11 @@ onMounted(() => {
       placeholder="1000"
       prependContent="$"
       @update:modelValue="formattedPrice" />
-    <CrushTextArea
-      v-model:value="plan.description"
+    <CrushEditor
+      v-model="plan.description"
       label="Descripción"
       placeholder="Agrega la descripción"
-      :max-length="maxLength"
-      @update:modelValue="descriptioInput" />
+      :max-length="maxLength" />
```

Comment:
The style guide has changed? 

let me know 
https://app.clickup.com/14221792/v/dc/dj0f0-2765

Reply:
good catch, thanks